addons/mail/static/src/models/rtc_outgoing_call/rtc_outgoing_call.js

```diff
@@ -0,0 +1,18 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one } from '@mail/model/model_field';
+
+registerModel({
+    name: 'RtcOutgoingCall',
+    identifyingFields: ['rtc', 'token'],
```

Comment:
As `rtc` is a messaging singleton, I don't think it's necessary to have it as identifying. It's like having `and messaging` as identifying on 100% of the models, sure that works, but that would be overkill.